test/wasm2js/left-to-right.2asm.js

```diff
@@ -972,7 +972,7 @@ function asmFunc(global, env, buffer) {
   $1$hi = i64toi32_i32$0;
   i64toi32_i32$0 = $0$hi;
   i64toi32_i32$1 = $1$hi;
-  FUNCTION_TABLE[i64_callee() | 0]($0, i64toi32_i32$0, $1, i64toi32_i32$1) | 0;
+  FUNCTION_TABLE[i64_callee() | 0 | 0]($0, i64toi32_i32$0, $1, i64toi32_i32$1) | 0;
```

Comment:
All does duplicate `| 0`  look a bit odd but expected I guess?

Reply:
That are expected in the unoptimized output files, yes - optimized files have `.opt`, and don't have those.